src/dynamic_honey_badger.rs

```diff
@@ -438,18 +609,41 @@ impl<Tx, NodeUid: Ord> Batch<Tx, NodeUid> {
         self.transactions.values().all(Vec::is_empty)
     }
 
-    /// Returns the change to the set of participating nodes, if any.
-    pub fn change(&self) -> Option<&Change<NodeUid>> {
-        self.change.as_ref()
+    /// Returns whether any change to the set of participating nodes is in progress or was
+    /// completed in this epoch.
+    pub fn change(&self) -> &ChangeState<NodeUid> {
+        &self.change
     }
 }
 
+/// An internal message that gets committed via Honey Badger to communicate synchronously.
```

Comment:
I find this sentence confusing. Maybe instead of referring to synchronous communication the comment could summarize the message content in a few words?